sources/packages/web/src/main.ts

```diff
@@ -80,6 +80,7 @@ import {
   faCopy as farCopy,
   faStickyNote,
   faCheckCircle as farCheckCircle,
+  faArrowAltCircleDown as farArrowAltCircleDown,
```

Comment:
We are no longer using the FontAwesome component. Please use the v-icon that is now supporting `Material Design Icons` and `Font Awesome`. There are some examples of both already on the solution.